app/views/admin/users/index/_controls.html.erb

```diff
@@ -19,7 +19,7 @@
       </div>
       <div class="crayons-field flex-row items-center gap-2">
         <%= f.label :role, "Filter by Role", class: "crayons-field__label whitespace-nowrap" %>
-        <%= f.select :role, options_for_select(Role::ROLES, params[:role]), { include_blank: true }, class: "crayons-select mt-0" %>
+        <%= f.select :role, options_for_select(Role::ROLES, params[:role]), { include_blank: "All roles" }, class: "crayons-select mt-0" %>
```

Comment:
This is for the current view (not behind the Feature Flag) that we're planning on removing once the new layout is complete.